example/android/app/build.gradle

```diff
@@ -28,8 +28,8 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "de.jonasbark.stripepaymentexample"
-        minSdkVersion 19
+        applicationId "jonasbark.de.stripepaymentexample"
```

Comment:
please revert this line - de.jonasbark is the correct format

Reply:
Sure I will revert it. But you have different package names throughout the example and it was only runnable with this packagename and not the provided one. 

Thats why I had to change the packagename to run in:
![image](https://user-images.githubusercontent.com/58904080/84477750-7071dd80-ac90-11ea-9538-04796b0ac969.png)

I have now fixed the packagename in the MainActivity. Now its working with your package name.

Reply:
Sure I will revert it. But you have different package names throughout the example and it was only runnable with this packagename and not the provided one.

Thats why I had to change the packagename to run in:
image

I have now fixed the packagename in the MainActivity. Now its working with your package name.